README.md

```diff
@@ -129,7 +129,7 @@ The step name is "openshiftDeploy".  Mandatory parameters are:
 
 Optional parameters are:
 
--  "waitTime":  Time in milliseconds to wait for deployment completion.  Default is 1 minute.
+-  "waitTime":  Time in milliseconds to wait for deployment completion.  Default is 10 minutes.
```

Comment:
@oatmealraisin - potential rebase hit unless we get you PR merged before this one.